develop/blocks/blockstates.md

```diff
@@ -2,7 +2,17 @@
 title: Block States
 description: Learn why blockstates are a great way to add visual functionality to your blocks.
 authors:
-  - IMB11
+  - imb11
+  - jamieswhiteshirt
+  - juuxel
```

Comment:
Again, this isnt from the wiki? This is from moddedmc.wiki which was original content - I am the currently the sole author of this page.